src/cloud-api-adaptor/podvm/Makefile.inc

```diff
@@ -36,6 +36,20 @@ AGENT_POLICY ?= yes
 AGENT_POLICY_PATH := $(FILES_DIR)/etc/kata-opa
 DEFAULT_AGENT_POLICY_FILE ?= allow-all.rego
 
+# Run github artifact provenance verifications
+VERIFY_PROVENANCE ?= no
+GUEST_COMPONENTS_REPO := confidential-containers/guest-components
+
+# probe version if the ref does not appear to be a git commit digest
+ifeq ($(shell echo $(GUEST_COMPONENTS_REF) | grep -E "[0-9a-f]{40}"),)
+	GUEST_COMPONENTS_COMMIT := $(shell curl \
+		--silent \
+		--retry 3 \
+		https://api.github.com/repos/$(GUEST_COMPONENTS_REPO)/commits/v$(GUEST_COMPONENTS_REF) \
+		| jq -e -r .sha)
+else
```

Comment:
Does this effectively mean that we can't use a guest-component build from a fork, unless we use the sha directly? I'm not sure that's a problematic restriction, just wanting to check my understanding.

Reply:
well, we have this chain of indirections from git reference => git digest => oci tag => oci digest since want to specify a human-friendly reference in versions.yaml.  

This chain needs to also work if you want to consume something from a fork, so you have to push something with the `resolved_to_sha($reference) + "_" + $arch` to the registry.

I think if this turns out to be a nuisance, there can be an "oci.digest" param in version yaml that will short circuit everything.